astring.cpp

```diff
@@ -118,22 +118,22 @@ AString & AString::operator=(const char *c)
 	return *this;
 }
 
-int AString::operator==(char *s)
+int AString::operator==(char *s) const
```

Comment:
clang didn't like comparing these and complained of ambiguity depending on which one was chose as the rhs/lhs.  This is the accepted way of doing this nowadays.